Description: compensate for expanded error message since Node.js 12
Origin: upstream, https://github.com/sindresorhus/resolve-from/commit/b5eefed
Author: Corey Farrell <git@cfware.com>
Bug: https://github.com/sindresorhus/resolve-from/issues/13
Last-Update: 2021-01-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test.js
+++ b/test.js
@@ -11,7 +11,7 @@
 		resolveFrom('fixture', './nonexistent');
 	}, Error);
 	t.is(moduleNotFoundError.code, 'MODULE_NOT_FOUND');
-	t.is(moduleNotFoundError.message, 'Cannot find module \'./nonexistent\'');
+	t.is(moduleNotFoundError.message.split(/[\n\r]+/)[0], 'Cannot find module \'./nonexistent\'');
 
 	const resolveFromfixture = resolveFrom.bind(null, 'fixture');
 	t.regex(resolveFromfixture('./fixture'), /fixture\/fixture\.js$/);
